### PR TITLE
feat(codmods): add /next codemod

### PR DESCRIPTION
**Pull request by talkor**
Decided to go for it anyway
https://monday.monday.com/boards/3532714909/pulses/7852782672

### CODE REVIEW

File: packages/codemod/transformations/core/v2-to-v3/__tests__/next-imports-migration.test.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import transform from "../next-imports-migration";
+import { defineInlineTest } from "jscodeshift/src/testUtils";
+
+describe("Rename type imports migration", () => {
```
[the remark this answers is not in the document]
Comment (talkor) [reply packages/codemod/transformations/core/v2-to-v3/__tests__/next-imports-migration.test.ts:4]: Done

File: packages/codemod/transformations/core/v2-to-v3/next-imports-migration.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { CORE_IMPORT_PATH } from "../../../src/consts";
+import { wrap, renameImportPath, getImports } from "../../../src/utils";
+import { TransformationContext } from "../../../types";
+
+/**
+ * 1. Changes imports from 'monday-ui-react-core/next' to 'monday-ui-react-core'
+ */
+function transform({ root }: TransformationContext) {
+  const imports = getImports(root, `${CORE_IMPORT_PATH}/next`);
```
Comment (YossiSaadi) [inline packages/codemod/transformations/core/v2-to-v3/next-imports-migration.ts:9]:
```suggestion
  const imports = getImports(root, `${CORE_NEXT_IMPORT_PATH}/next`);
```

File: packages/codemod/transformations/core/v2-to-v3/next-imports-migration.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { CORE_IMPORT_PATH } from "../../../src/consts";
```
Comment (YossiSaadi) [inline packages/codemod/transformations/core/v2-to-v3/next-imports-migration.ts:1]:
```suggestion
import { CORE_NEXT_IMPORT_PATH } from "../../../src/consts";
```

**Review by YossiSaadi — COMMENTED** [review]
are you sure this should be part of v3 default migration?

**Review by YossiSaadi — APPROVED** [review]
My bad, understood now

File: packages/codemod/transformations/core/v2-to-v3/next-imports-migration.ts (hunk 3 above)
Comment (talkor) [reply packages/codemod/transformations/core/v2-to-v3/next-imports-migration.ts:1]: It should be renamed to `monday-ui-react-core` (which be later on changes to `@vibe/core`)

Comment (YossiSaadi) [reply packages/codemod/transformations/core/v2-to-v3/next-imports-migration.ts:1]: oh right

**Pull request merged**